### DISCUSSION TITLE
General Peeves

### DISCUSSION THREAD

**Discussion by Theedoo** [ppy/osu#22373] [Ideas]
This is more of a "things I've noticed, and seem to be slightly off-putting, with ideas of how to change it"

1. The entire screen feeling "too zoomed in." An easy fix would be to just set the default zoom to something a bit lower, or asking in the "setup" if they would prefer a "Small, Medium, Large" size of screen
2. The song thumbnail taking up much more space than it (subjectively) should, an easy(?) solve would be to allow us to disable it in the settings, with the option to unblur the background
3. Skin layout editor not having snapping, locked ratio scaling, cropping, and _un_-locked ratio scaling
4. Better/More accurate number input for tablet areas (as well as ability to click and drag tablet area, and have it easily locked to playable area,) or just in general
5. pressing ESC upon reaching the score screen immediately "Throws you out of it" as opposed to stable's
   a. tap ESC while within fade-in time
   b. it cuts the fade-in, "instantly" putting you to the score screen instead of "properly" exiting

**Comment by mk56-spn** [comment]
Many of these are known already, also opening discussions which bunch things together into one post is generally frowned upon. Since it makes handling them a pain , I'd suggest searching thoroughly to make sure each one of these hasn't come up before. And then opening separate discussions where applicable
